config/development.yml.dist

```diff
@@ -68,3 +68,12 @@ security:
 reviewer:
   users:
     -
+
+opencfpcentral:
```

Comment:
When first running

```
$ make
```

after checkout, the process failed with

```
  The service "OpenCFP\Http\Action\Security\CentralAction" has a dependency on a non-existent parameter "opencfpcentral.clientId".
```

Maybe we should extract parameters and use [`incenteev/parameter-handler`](https://github.com/Incenteev/ParameterHandler)? This is clearly out of scope of this PR, though.

Reply:
Make now passes for everything I've done here (saw similar errors but thanks for pointing out I forgot to run it) and I am open to using that parameter-handler but in a follow-up PR once someone other than me can get SSO up-and-running for themselves.